lib/AST/ASTScope.cpp

```diff
@@ -617,27 +617,51 @@ ASTScope *ASTScope::createRoot(SourceFile *sourceFile) {
   return scope;
 }
 
+// FIXME: This is a vestige of multiple parameter lists.
+static ParamDecl *getParameter(AbstractFunctionDecl *func,
```

Comment:
@DougGregor I feel really guilty about this :( Sorry about that. I hope one of us finds time to work on ASTScope one day!